tools/testing/requirements.txt

```diff
@@ -64,61 +62,50 @@ coverage==5.5 \
     --hash=sha256:f0b278ce10936db1a37e6954e15a3730bea96a0997c26d7fee88e6c396c2086d \
     --hash=sha256:f11642dddbb0253cc8853254301b51390ba0081750a8ac03f20ea8103f0c56b6
     # via
-    #   -r tools/testing/requirements.txt
+    #   -r tools/testing/requirements.in
     #   pytest-cov
 iniconfig==1.1.1 \
     --hash=sha256:011e24c64b7f47f6ebd835bb12a743f2fbe9a26d4cecaa7f53bc4f35ee9da8b3 \
     --hash=sha256:bc3af051d7d14b2ee5ef9969666def0cd1a000e121eaea580d4a313df4b37f32
-    # via
-    #   -r tools/testing/requirements.txt
-    #   pytest
+    # via pytest
 packaging==21.0 \
     --hash=sha256:7dc96269f53a4ccec5c0670940a4281106dd0bb343f47b7471f779df49c2fbe7 \
     --hash=sha256:c86254f9220d55e31cc94d69bade760f0847da8000def4dfe1c6b872fd14ff14
-    # via
-    #   -r tools/testing/requirements.txt
-    #   pytest
-pluggy==1.0.0 \
-    --hash=sha256:74134bbf457f031a36d68416e1509f34bd5ccc019f0bcc952c7b909d06b37bd3 \
-    --hash=sha256:4224373bacce55f955a878bf9cfa763c1e360858e330072059e10bad68531159
-    # via
-    #   -r tools/testing/requirements.txt
-    #   pytest
+    # via pytest
+pluggy==0.13.1 \
```

Comment:
It looks like we downgraded the `pluggy` version here..

Reply:
i think that is required - there was a conflict with pytest, and a hurried fix this week - ill check again tho

Reply:
yep, re-running pip-compile gives the ~same result, so i think that is the current deps situation

Reply:
my bad - i wasnt upgrading when running pip-compile - ive reverted the pluggy downgrade